src/cli/index.js

```diff
@@ -0,0 +1,109 @@
+import qrcode from 'qrcode-terminal';
+import { Promise } from 'es6-promise';
+import chalk from 'chalk';
+import { getInstallations as getBrowserInstallations } from 'testcafe-browser-natives';
+import CliArgumentParser from './argument-parser';
+import createTestCafe from '../';
+
+const ERR_PREFIX = chalk.red('ERROR ');
+
+function exit (code) {
+    // NOTE: give process time to flush output.
+    // It's necessary for some environments.
+    setTimeout(() => process.exit(code), 0);
```

Comment:
We should increase the timeout to close browsers after the run. It works with about 1000 ms on my machine. It's a time to go to the idle page and close the browser.
TestCafe uses the broken vesion of `testcafe-browser-natives`, so to test this I've put the working `testcafe-browser-natives` module version to `node_modules`